src/include/sof/drivers/edma.h

```diff
@@ -91,4 +91,11 @@ int edma_init(void);
 #define trace_edma_error(format, ...) trace_error(TRACE_CLASS_DMA,\
 						  format, ##__VA_ARGS__)
 
+#define EDMA0_ESAI_CHAN_RX	6
+#define EDMA0_ESAI_CHAN_TX	7
+#define EDMA0_CHAN_MAX		32
+
+#define EDMA0_ESAI_CHAN_RX_IRQ	442
+#define EDMA0_ESAI_CHAN_TX_IRQ	442
+
 #endif /* __SOF_DRIVERS_EDMA_H__ */
```

Comment:
Do you think you could add SAI numbers too?
Would it make sense to you to do it now?

Reply:
@groncarolo @jlacla please confirm that this is correct:

```
#define EDMA0_ESAI_CHAN_RX	14
#define EDMA0_ESAI_CHAN_TX	15
#define EDMA0_SAI_CHAN_RX_IRQ	349
#define EDMA0_SAI_CHAN_TX_IRQ	349
```

Reply:
Shoudl be these

`#define EDMA0_SAI_RX_CHAN       14`
`#define EDMA0_SAI_TX_CHAN       15`

`#define EDMA0_SAI_CHAN_RX_IRQ           349`
`#define EDMA0_SAI_CHAN_TX_IRQ           349`

Reply:
To make it consistent I made them with _CHAN_RX and _CHAN_TX instead of _RX_CHAN and _TX_CHAN.